run

```diff
@@ -24,8 +24,6 @@ while (( $# > 0 )); do
   djbuild ) build_django ;;   # download and build all images, migrate database
   webpack ) docker-compose run --rm webpack ${rest:-bash}; exit;;  # login to webpack container as default user
   build   ) webpack_build ;; # build css+js client bundles and express bundle;
-  npmi    ) docker-compose run --rm webpack install $rest; exit ;; # npm install [packages...]
-  npmup   ) docker-compose run --rm webpack update ;; # npm update js packages
```

Comment:
Remove these commands, as they are better done manually. 

`$ ./run webpack bash`

Then in docker container 

`$ npm update` or `$ npm install [packages]`

Before commiting changes, run tests and try building using `npm run test` and `npm run build`. This will break if the new packages are not compatible.